tests/dataset_runner.py

```diff
@@ -215,7 +215,7 @@ def _assert_data_browser_bundles(self, project_shortname):
         try:
             response_json = self.azul_agent.get_specimen_by_project(project_shortname)
             orange_uuids = {bundle["bundleUuid"] for hit in response_json["hits"] for bundle in hit["bundles"]}
-            assert orange_uuids == set(self.primary_bundle_uuids)
+            assert orange_uuids == set().union(self.primary_bundle_uuids, self.secondary_bundle_uuids)
```

Comment:
I would have preferred either of these:

`{*self.primary_bundle_uuids, *self.secondary_bundle_uuids}`
`set(self.primary_bundle_uuids) | set(self.primary_bundle_uuids)`
`set(self.primary_bundle_uuids).union(self.primary_bundle_uuids)`